src/Client/BuzzHouse/Utils/MD5Impl.h

```diff
@@ -17,19 +20,7 @@ class MD5Impl
     uint8_t input_buffer[input_buffer_size];
 
 public:
-    void hashFile(const std::string & file_path, uint8_t digest[16])
-    {
-        std::ifstream file(file_path, std::ios::binary);
-
-        if (!file)
-        {
-            throw std::runtime_error("Could not open file: " + file_path);
-        }
-        MD5_Init(&ctx);
-        while (file.read(reinterpret_cast<char *>(input_buffer), input_buffer_size) || file.gcount() > 0)
-        {
-            MD5_Update(&ctx, reinterpret_cast<const unsigned char *>(input_buffer), file.gcount());
-        }
-        MD5_Final(digest, &ctx);
-    }
+    void hashFile(const std::string & file_path, uint8_t digest[16]);
```

Comment:
Why are we not using `Poco::MD5Engine` with a `Poco::FileInputStream`? Why do we use OpenSSL directly?

Reply:
I can't find `Poco::FileInputStream` examples with Digest engines. Can you point me one?

Reply:
You're right. I can't see any way a `Poco::MD5Engine` takes a `Poco::FIleInputStream`. I don't know if I misread when I wrote the comment or I assumed it would exist :roll_eyes: